package-native-zip.sh

```diff
@@ -103,6 +103,23 @@ echo "------------------------------------------------"
 echo ""
 zip -j dist/$PACKAGE_NAME dist/conduct dist/sandbox dist/shazar
 
+if [ "$UNAME_VALUE" = "Darwin" ]; then
+    echo "------------------------------------------------"
+    echo "Commands to create the Homebrew package pull request for $PACKAGE_NAME"
+    echo "------------------------------------------------"
+    echo ""
+
+    echo "Checkout the repo at https://github.com/typesafehub/homebrew-conductr and then issue"
```

Comment:
I don't think you need to have the repo cloned, but you _do_ need to have it "tapped" in your Homebrew installation, so I'd say: "Run `brew tap typesafehub/conductr` and then issue"...